src/jvm/main/org/jetbrains/kotlinx/lincheck/TraceDebugger.kt

```diff
@@ -10,4 +10,85 @@
 
 package org.jetbrains.kotlinx.lincheck
 
+import org.jetbrains.kotlinx.lincheck.execution.ExecutionResult
+import org.jetbrains.kotlinx.lincheck.execution.ExecutionScenario
+import org.jetbrains.kotlinx.lincheck.execution.threadsResults
+import org.jetbrains.kotlinx.lincheck.strategy.managed.forClasses
+import org.jetbrains.kotlinx.lincheck.strategy.managed.modelchecking.ModelCheckingOptions
+import org.jetbrains.kotlinx.lincheck.strategy.managed.modelchecking.constructTraceForPlugin
+import org.jetbrains.kotlinx.lincheck.transformation.LincheckJavaAgent.ensureClassHierarchyIsTransformed
+import org.jetbrains.kotlinx.lincheck.verifier.Verifier
+import java.lang.reflect.Method
+import java.lang.reflect.Modifier
+import kotlin.reflect.jvm.kotlinFunction
+
 val isInTraceDebuggerMode = System.getProperty("lincheck.traceDebuggerMode", "false").toBoolean()
+
+internal object TraceDebuggerInjections {
+    val classUnderTimeTravel: String? = System.getProperty("traceDebugger.className")
+    val methodUnderTimeTravel: String? = System.getProperty("traceDebugger.methodName")
```

Comment:
Let's rename `classUnderTimeTravel` and `methodUnderTimeTravel` appropriately, since we are dropping time traveling terminology.